exasol_integration_test_docker_environment/lib/test_environment/spawn_test_database.py

```diff
@@ -336,8 +337,8 @@ def _add_exa_conf(
         certificate_dir = CERTIFICATES_MOUNT_DIR if self.certificate_volume_name is not None \
                             else CERTIFICATES_DEFAULT_DIR
         template_file = self._db_file("EXAConf")
-        template = Template(template_file.read_text())
-        additional_db_parameter_str = " ".join(self.additional_db_parameter)
+        template = Template(template_file.read_text()) # type: ignore
```

Comment:
Why does this need ignore, maybe we should add reason to ignores that are not obvious

Reply:
Type ignore could be removed